src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -134,5 +121,13 @@ protected string Resolve(string name)
 
             return _config.HostConfig.NameResolver.ResolveWholeString(name);
         }
+
+        internal static void AddStorageAccountAttribute(Collection<CustomAttributeBuilder> attributes, string connection)
```

Comment:
Would Script ever add a StorageAccountAttribute on anything besides the parameter?  

---
In reply to: [56770291](https://github.com/Azure/azure-webjobs-sdk-script/pull/128#discussion_r56770291) [](ancestors = 56770291)